server/monitor.go

```diff
@@ -1101,13 +1101,18 @@ type LeafNodeOptsVarz struct {
 	Remotes     []RemoteLeafOptsVarz `json:"remotes,omitempty"`
 }
 
+// Contains lists of subjects not allowed to be imported/exported through a leaf node connection
+type RemoteLeafOptDeny struct {
```

Comment:
I was wondering if we should make this very generic, so that if in the future we need an exports/imports []string, we can use in different places. That being said, I have no idea how I would name this new struct.
Also, there is a way to do type aliases...

Reply:
how does DenyRules sound?

Reply:
Sounds good to me.

Reply:
renamed